### DISCUSSION TITLE
"pages build and deployment" workflow weirdness

### DISCUSSION THREAD

**Discussion by patreeceeo** [community/community#47022] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The index page for my site loads fine until I set up a custom domain, then I get a 404 on the index.

Here's the Pages API GET response:
```json
curl \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer $GH_TOKEN"\   
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/repos/patreeceeo/patreeceeo.github.io/pages
{
  "url": "https://api.github.com/repos/patreeceeo/patreeceeo.github.io/pages",
  "status": "built",
  "cname": "www.codebaser.net",
  "custom_404": false,
  "html_url": "https://www.codebaser.net/",
  "build_type": "legacy",
  "source": {
    "branch": "main",
    "path": "/docs"
  },
  "public": true,
  "protected_domain_state": "verified",
  "pending_domain_unverified_at": null,
  "https_certificate": {
    "state": "approved",
    "description": "The certificate has been approved.",
    "domains": [
      "www.codebaser.net",
      "codebaser.net"
    ],
    "expires_at": "2023-05-11"
  },
  "https_enforced": true
}
```
Before I set up the custom domain, I could go to https://patreeceeo.github.io/ and got 200, but after setting up the above custom domain, both https://www.codebaser.net and https://codebaser.net 404.

How could this be?

A guess: Something about my [Workflow](https://github.com/patreeceeo/patreeceeo.github.io/blob/main/.github/workflows/pages.yml)?

**Comment by jge162** [comment]
Try changing the source path to the root directory of your repository, which is typically "/", and see if that fixes the 404 error

**Reply by patreeceeo** [reply]
still get a 404 for both https://codebaser.net and https://codebaser.net/docs, as well as for www subdomain.

**Comment by mfdmh-haus** [comment]
No answers yet. I've looked into this a bit more and I suspect it's the built-in "pages build and deployment" workflow that's the root cause. I created a brand new account and now the 404 happens even before trying to use a custom domain. My custom workflow produces the correct output, which I verified by downloading the ZIP archive artifact. But when I do the same for the built-in workflow, I'm seeing a directory structure and code that definitely is not mine. For instance, I see a style.css that contains a minified normalize.css. I'm not using normalize and I'm not currently minifying code.

I have made sites with github pages before and haven't had these issues. I'm going to host elsewhere, at least for now. I would like to host on GitHub but seems it's just not in the cards for me right now. I'd be happy to provide more details if a support person ever looks at this.

**Reply by mfdmh-haus** [reply]
Oh yeah, this is my other account. Forgot I was still signed in here. This is @patreeceeo

**Comment by patreeceeo** [comment]
Seeing the same weird output on my own account https://github.com/patreeceeo/patreeceeo.github.io/actions/runs/4169587214. You can look through the codebase, there's no normalize.css.

**Comment by patreeceeo** [comment]
Got it deployed, running and successfully custom domain'd (https://codebaser.net/) on Deno Deploy. I actually found the process there to be easier than on GitHub Pages.
